mriqc/interfaces/diffusion.py

```diff
@@ -27,6 +27,8 @@
 import nibabel as nb
 import numpy as np
 import scipy.ndimage as nd
+from dipy.core.gradients import gradient_table
+from dipy.stats.qc import find_qspace_neighbors
```

Comment:
If we're going to depend on a new feature, we need to set the minimum version in `pyproject.toml`.

Reply:
:eagle: Thanks. Done in a3bdc5d. Force pushed.

Reply:
OK, so this conflicts with Python 3.8, as DIPY has dropped support for Python 3.8 following SPEC-0000 of Scientific Python:
https://scientific-python.org/specs/spec-0000/#support-window

Should we drop support for Python 3.8?

Reply:
Yes, we should drop 3.8, as it's EOL. If we follow SPEC0, we should be dropping 3.10.

TBH, I'm fine with this for end-user programs. I would probably be more conservative with nireports, since it has more potential for external reuse, but pretty much everything else could go to SPEC0.

Reply:
I do not know if we follow SPEC0 strictly; for now, I am dropping 3.8. Maybe this can be discussed on the next Tech meeting, have it clarified in the README or across the NiPreps (with the necessary exceptions), and enforce it as time permits.